make/photon/common/install_cert.sh

```diff
@@ -2,17 +2,21 @@
 
 set -e
 
-if [ ! -f /etc/pki/tls/certs/ca-bundle.crt.original ]; then
-    cp /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.crt.original
+if [ ! -f ~/ca-bundle.crt.original ]; then
+    cp /etc/pki/tls/certs/ca-bundle.crt ~/ca-bundle.crt.original
 fi
 
-if [ -f /harbor_cust_cert/custom-ca-bundle.crt ]; then
+cp ~/ca-bundle.crt.original /etc/pki/tls/certs/ca-bundle.crt
+
+if [ "$(ls -A /harbor_cust_cert)" ]; then
     if grep -q "Photon" /etc/lsb-release; then
-        echo "Appending custom ca bundle ..."
-        cp /etc/pki/tls/certs/ca-bundle.crt.original /etc/pki/tls/certs/ca-bundle.crt
-        cat /harbor_cust_cert/custom-ca-bundle.crt >> /etc/pki/tls/certs/ca-bundle.crt
+        for z in /harbor_cust_cert/*; do
```

Comment:
Could we add more sanity check  to `z`?
for example it has to be a file and it has to match some pattern?